modules/tinymce/src/themes/silver/main/ts/ui/view/View.ts

```diff
@@ -73,6 +73,15 @@ const renderViewHeader = (spec: ViewHeaderSpec) => {
         ...(isPhone || isTablet ? [ 'tox-view--mobile', 'tox-view--scrolling' ] : [])
       ]
     },
+    behaviours: Behaviour.derive([
+      Focusing.config({}),
+      Keying.config({
+        mode: 'flow',
+        selector: 'button, .tox-button',
+        focusInside: FocusInsideModes.OnEnterOrSpaceMode,
+        cycles: false,
```

Comment:
Any particular reason why this doesn't cycle? I think other toolbars and the dialogs cycle on tab.

Reply:
actually there is no particular reason, I removed it